packages/flutter_tools/test/general.shard/ios/simulators_test.dart

```diff
@@ -901,6 +901,24 @@ Dec 20 17:04:32 md32-11-vm1 Another App[88374]: Ignore this text'''
         throwsToolExit(message: r'Unable to launch'),
       );
     });
+
+    testWithoutContext('.stopApp() handles exceptions', () async {
```

Comment:
is there a way to test that the app is actually terminated?

Reply:
Do you mean in an integration test?  Let me see if I can get something working...

Reply:
Added a check to the hot reload tests.